src/Microsoft.OData.Edm/EdmUtil.cs

```diff
@@ -318,6 +318,8 @@ internal static string FullyQualifiedName(IEdmVocabularyAnnotatable element)
                 else
                 {
                     IEdmProperty property = element as IEdmProperty;
+                    IEdmOperationParameter parameter = element as IEdmOperationParameter;
+                    IEdmEnumMember enumMember = element as IEdmEnumMember;

```

Comment:
Since casts can be relatively expensive, probably better to try each one in turn, and only attempt cast if the previous cast fails.

Reply:
@mikepizzo Changed the codes. Thanks.